lib/waipu_api.py

```diff
@@ -192,4 +192,21 @@ def play_recording(self, id):
     def open_eu_network(self):
         self.get_token()
         url = "https://eunet.waipu.tv/api/open-eu-network"
-        return requests.post(url, headers=self.prepare_headers()).status_code
\ No newline at end of file
+        return requests.post(url, headers=self.prepare_headers()).status_code
+
+    def deleteRecordings(self, recordId):
```

Comment:
we should either rename it to `deleteRecording` or make parameter `recordId` a list, to accept multiple recordings to be deleted